tests/api/v1/datadog/api_service_level_objectives_test.go

```diff
@@ -627,6 +627,11 @@ func TestSLOCorrectionsLifecycle(t *testing.T) {
 	defer finish()
 	assert := tests.Assert(ctx, t)
 
+	Client(ctx).GetConfig().SetUnstableOperationEnabled("CreateSLOCorrection", true)
+	Client(ctx).GetConfig().SetUnstableOperationEnabled("GetSLOCorrection", true)
+	Client(ctx).GetConfig().SetUnstableOperationEnabled("UpdateSLOCorrection", true)
+	Client(ctx).GetConfig().SetUnstableOperationEnabled("DeleteSLOCorrection", true)
+
```

Comment:
```suggestion
	Client(ctx).GetConfig().SetUnstableOperationEnabled("ListSLOCorrection", true)
```
I think we're just missing List :)